table/plain_table_reader.cc

```diff
@@ -114,13 +114,11 @@ PlainTableReader::~PlainTableReader() {
 
 Status PlainTableReader::Open(const ImmutableCFOptions& ioptions,
                               const EnvOptions& env_options,
+                              const PlainTableOptions& table_options,
                               const InternalKeyComparator& internal_comparator,
-                              unique_ptr<RandomAccessFile>&& file,
+                              std::unique_ptr<RandomAccessFile>&& file,
                               uint64_t file_size,
-                              unique_ptr<TableReader>* table_reader,
-                              const int bloom_bits_per_key,
-                              double hash_table_ratio, size_t index_sparseness,
-                              size_t huge_page_tlb_size, bool full_scan_mode) {
```

Comment:
I like the change of passing through PlainTableOptions all the way down. But is it related to the port?